packs/outlaws-of-alkenstar-bestiary/smog-giant.json

```
@@ -287,7 +287,17 @@
                     "remaster": false,
                     "title": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "definition": [
+                            "ingested",
+                            "poison"
```

```suggestion
                            "item:trait:ingested",
                            "item:trait:poison"
```
Always, always, always avoid using top-level roll options.